README.md

```diff
@@ -1,7 +1,82 @@
 # Open MCT
 
-Open MCT is a web-based platform for mission operations user interface
-software.
+Open MCT is a next-generation mission control framework being developed at the NASA Ames Research Center in Silicon Valley. Web-based, for desktop and mobile.
+
+Please visit our [Official Site](https://nasa.github.io/openmct/)
+
+##How is NASA Using Open MCT?
+
+Software based on Open MCT is being used for mission planning and operations in the lead up to the [Resource Prospector mission](https://www.nasa.gov/resource-prospector) and at NASA's Jet Propulsion Laboratory to view data from the Curiosity Rover.
+
+##Features
+
+ - Support planning and operations of any system that produces telemetry
+ -  Support space missions
+ - Visualize Data
+   - Streaming Data
+   - Historical Data
+   - Imagery
+   - Timelines
+   - Procedures
+   - etc.
+
+##See Open MCT in Action
+
+Do you want to see how Open MCT work? Try our demo.
```

Comment:
Could we rephrase - To try OpenMCT for yourself, take a look at our online demo.